rust/pst/contract/implementation/src/contract.rs

```diff
@@ -1,14 +1,26 @@
 use async_recursion::async_recursion;
-
-use warp_pst::action::{Action, ActionResult};
-use warp_pst::state::State;
-
-use crate::actions::{Actionable, *};
-use warp_wasm_utils::contract_utils::js_imports::{Block, Contract, log, SmartWeave, Transaction};
-
+use warp_pst::{
+    action::{Action, ViewResponse},
+    error::ContractError,
+    state::State,
+};
+
+use crate::actions::*;
+use warp_wasm_utils::{
+    contract_utils::{
+        handler_result::ActionResult,
+        js_imports::{SmartWeave, Transaction},
+    },
+    warp_bindgen,
+};
+
+#[warp_bindgen]
 #[async_recursion(?Send)]
-pub async fn handle(state: State, action: Action) -> ActionResult {
-    let original_caller = Transaction::owner();
+pub async fn handle2(
```

Comment:
is it still prohibited to use the 'handle' name in the contract itself?

Reply:
It is allowed. I'll fix it. Our code is generated into a nested scope and exported out of WASM from there so it does not interfere with anything written by the contract author.